CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ This file is used to list changes made in each version of the Java cookbook.
 
 - Upgrade circleci orb to version 2 and add yamllint and markdown lint
 
+## 4.2.0 - 2019-06-20
+
+- Fix for issue 538
+- Added "download_path" node attribute defaulting to file_cache_path
+- Replaced all hardcoded instances of file_cache_path with the node attribute 
```

Comment:
markdown linter is complaining of extra trailing spaces here

Reply:
fyi, you can download and run locally https://github.com/markdownlint/markdownlint `mdl .`